src/goto-programs/remove_function_pointers.cpp

```diff
@@ -355,14 +330,56 @@ void remove_function_pointerst::remove_function_pointer(
     }
   }
 
-  remove_function_pointer(goto_program, function_id, target, functions);
+  ::remove_function_pointer(
+    message_handler,
+    symbol_table,
+    goto_program,
+    function_id,
+    target,
+    functions,
+    add_safety_assertion);
 }
 
-void remove_function_pointerst::remove_function_pointer(
+static std::string function_pointer_assertion_comment(
+  const std::unordered_set<symbol_exprt, irep_hash> &candidates)
+{
+  std::stringstream comment;
+
+  comment << "dereferenced function pointer at must be ";
```

Comment:
Needs location after `at`?

Reply:
Done, thanks! (Well, I actually removed the location.)